packages/babel-preset-env/test/fixtures/debug/shippedProposals-chrome-80/stdout.txt

```diff
@@ -11,21 +11,12 @@ Using plugins:
   transform-unicode-sets-regex { chrome < 112 }
   transform-class-static-block { chrome < 94 }
   transform-private-property-in-object { chrome < 91 }
-  syntax-class-properties
   transform-private-methods { chrome < 84 }
-  syntax-numeric-separator
   transform-logical-assignment-operators { chrome < 85 }
-  syntax-nullish-coalescing-operator
-  syntax-optional-chaining
-  syntax-json-strings
-  syntax-optional-catch-binding
-  syntax-async-generators
-  syntax-object-rest-spread
+  transform-export-namespace-from { }
   bugfix/transform-v8-spread-parameters-in-optional-chaining { chrome < 91 }
   transform-modules-commonjs
   transform-dynamic-import
-  transform-export-namespace-from { }
-  syntax-import-meta
   syntax-import-attributes
```

Comment:
These in the `debug` folder are Babel 8 tests, I'm not sure about why they included the syntax plugins but 🤷 the new output is correct